Source/SwiftLintBuiltInRules/Rules/Idiomatic/AnonymousArgumentInMultilineClosureRule.swift

```diff
@@ -8,6 +8,23 @@ struct AnonymousArgumentInMultilineClosureRule: OptInRule {
         identifier: "anonymous_argument_in_multiline_closure",
         name: "Anonymous Argument in Multiline Closure",
         description: "Use named arguments in multiline closures",
+        rationale: """
+        In multiline closures, for clarity, prefer using named arguments
+
+        ```
+            closure { arg in
+                print(arg)
+            }
```

Comment:
The indentation leads to indentation in the rendered Markdown as well. We should avoid that and rather indent the blocks before it's printed to the console.